Documentation/contributing.rst

```diff
@@ -1274,76 +1560,279 @@ simplify our Jenkinsfiles. The exported methods are:
-4. The first commit in the release branch is to change the version to
-   ``X.Y+1.0``.
-5. The next commit goes into the master branch and sets the version to
-   ``X.Y+1.90`` to ensure that the master branch will be considered more recent
-   than any stable release of the major release that is about to be published.
+.. _stable_releases:
 
 Stable releases
 ~~~~~~~~~~~~~~~
 
-The committers can nominate commits pushed to the master as stable release
-candidates in which case they will be backported to previous release branches.
-Upon necessity, stable releases are published with the version ``X.Y.Z+1``.
+The committers can nominate PRs merged into master as required for backport
+into the stable release branches. Upon necessity, stable releases are published
+with the version ``X.Y.Z+1``. Stable releases are regularly released in high
+frequency or on demand to address major incidents.
+
+In order to guarantee stable production usage while maintaining a high release
+cadence, the following stable releases will be maintained:
 
-Criteria for the inclusion into stable release branches are:
+* Stable backports into the last two releases
+* :ref:`lts` release for extended long term backport coverage
+
+
+Backport criteria for X.Y.Z+n
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Criteria for the inclusion into latest stable release branch, i.e. what goes
+into ``v1.1.x`` before ``v1.2.0`` has been released:
+
+- All bugfixes
+
+Backport criteria for X.Y-1.Z
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Criteria for the inclusion into the stable release branch of the previous
+release, i.e. what goes into ``v1.0.x``, before ``v1.2.0`` has been released:
 
 - Security relevant fixes
 - Major bugfixes relevant to the correct operation of Cilium
 
+.. _lts:
 
-Steps to release
-~~~~~~~~~~~~~~~~
+LTS
+~~~
+
+The group of committers nominates a release to be a long term stable release.
+Such releases are guaranteed to receive backports for major and security
+relevant bugfixes. LTS releases will be declared end of life after 6 months.
+The group of committers will nominate and start supporting a new LTS release
+before the current LTS expires. If for some reason, no release can be declared
+LTS before the current LTS release expires, the current LTS lifetime will be
+extended.
+
+Given the current 6 weeks release cadence, the development teams will aim at
+declaring every 3rd release to be an LTS to guarantee enough time overlap
+between LTS release.
+
+Current LTS releases
+^^^^^^^^^^^^^^^^^^^^
+
++----------------------+---------------------------+-----------------------+
+| Release              | Original Release Date     | Scheduled End of Life |
++======================+===========================+=======================+
+| 1.0                  | 2018-04-24                | 2018-10-24            |
++----------------------+---------------------------+-----------------------+
+
+.. _generic_release_process:
+
+Generic Release Process
```

Comment:
For major releases, we should also apply the change to CODEOWNERS to remove most of the owners.

I think that the ones that should remain are:
```
* @cilium/janitors
api/ @cilium/api
monitor/payload @cilium/api
pkg/apisocket/ @cilium/api
pkg/policy/api/ @cilium/api
pkg/proxy/accesslog @cilium/api
```

Reply:
Good point. Added